libs/boost/Makefile

```diff
@@ -256,8 +256,8 @@ define Package/boost/config
 			prompt "Boost $(lib) library."
 			default m if ALL
 			$(if $(findstring locale,$(lib)),depends on BUILD_NLS,)\
-			$(if $(findstring python,$(lib)),depends on PACKAGE_$(lib),)
-
+			$(if $(findstring python,$(lib)),depends on PACKAGE_$(lib),)\
+			$(if $(findstring fiber,$(lib)),depends on !TARGET_brcm63xx,)
```

Comment:
Try something like this:
```
                      # Don't build Boost.fiber if mips32r1
                      $(if $(findstring fiber,$(lib)),depends on CPU_TYPE!=mips32 && CPU_TYPE!=mips64,) \

Reply:
Ok, I have to apologise to you for over-complicating the solution.
I didn't notice I could compare those variables and used them like that.

The Makefile is now disabling Fiber for all targets where CPU_TYPE is specifically mips32 or mips64.